src/components/footer.astro

```diff
@@ -89,5 +92,7 @@ import { EPSLogo } from "./logo/eps-logo";
         </p>
       </div>
     </article>
+
+    <p class="mb-4" style="color: rgba(255, 255, 255, 0.4)">version: {gitVersion} @ {buildTimestamp}</p>
```

Comment:
```suggestion
          <p class="mb-4" style="color: rgba(255, 255, 255, 0.4)">version: {gitVersion} @ {buildTimestamp}</p>
      </div>
    </article>

```

Reply:
Was better but I want to be able to find it. :)

Reply:
Yes, I think I messed up the previous merge and only did the mb-4 without moving the tag :D